types/recharts/index.d.ts

```diff
@@ -163,7 +164,7 @@ export interface AreaProps extends EventAttributes, Partial<PresentationAttribut
     connectNulls?: boolean;
     activeDot?: boolean | object | React.ReactElement<any> | ContentRenderer<any>;
     dot?: boolean | object | React.ReactElement<any> | ContentRenderer<DotProps>;
-    label?: boolean | object | React.ReactElement<any> | LabelProps['content'];
+    label?: boolean | object | ContentRenderer<any> | React.ReactElement<any> | LabelProps['content'];
```

Comment:
Why the `ContentRenderer<any>`? `LabelProps['content']` includes `ContentRenderer<Label>`, isn't that enough already? Maybe we could even remove `React.ReactElement<any>` here, since that is also included in `LabelProps['content']`

Reply:
Please read the docs I referred to. You can pass a function which returns a Node as label prop. This is conform spec, and currently not implemented in these definitions, hence the addition.

Reply:
@mthmulders I see what you mean with LabelProps['content'] by the way, but TypesSript throws an error with this approach. We could be better off removing LabelProps['content'] here.

Reply:
I've read those docs before I asked and I didn't understand them. That's exactly why I asked. I understand you can pass a function - but as far as I can see, `LabelProps['content']` contains exactly that option. It even uses the same type. So why is the `ContentRenderer` needed here?

Reply:
Ah, well, maybe we're over-asking the tsc powers here... Could you check what happens if we remove `LabelProps['content']` here? Would it still work?

Reply:
Hehe yeah that could be the thing here. I've tested removing the `LabelProps['content']` typing and it seems to work fine, so I've commited it! Please review :-)

Reply:
Great, thanks a lot!